ssl/ssl_x509.cc

```diff
@@ -765,6 +765,14 @@ void SSL_CTX_set_cert_store(SSL_CTX *ctx, X509_STORE *store) {
   ctx->cert_store = store;
 }
 
+void SSL_CTX_set1_cert_store(SSL_CTX *ctx, X509_STORE *store) {
+  check_ssl_ctx_x509_method(ctx);
```

Comment:
`check_ssl_ctx_x509_method` succeeds when `ctx` is null, [it only ensures](https://github.com/aws/aws-lc/blob/2352c50a6b5b95c12aed8b92c5881def3218a646/ssl/ssl_x509.cc#L169) when it's not null that `ctx-.x509_method` is set correctly.

Reply:
Yes it's also called at the start of every publicly accessible function in this file.